acceptance/acceptance_test.go

```diff
@@ -988,3 +995,85 @@ func prepareWheelBuildDirectory(t *testing.T, dir string) string {
 
 	return latestWheel
 }
+
+func applyBundleConfig(t *testing.T, tmpDir string, bundleConfig map[string]any) string {
+	validConfig := make(map[string]map[string]any, len(bundleConfig))
+
+	for _, configName := range utils.SortedKeys(bundleConfig) {
+		configValue := bundleConfig[configName]
+		if configValue == "" {
+			continue
+		}
+		// either "" or a map are allowed
+		cfg, ok := configValue.(map[string]any)
+		if !ok {
+			t.Fatalf("Unexpected type for BundleConfig.%s: %#v", configName, configValue)
+		}
+		validConfig[configName] = cfg
+	}
+
+	if len(validConfig) == 0 {
+		return ""
+	}
+
+	var configPath, configData string
+	filenames := []string{"databricks.yml", "databricks.yml.tmpl"}
+
+	for _, filename := range filenames {
+		path := filepath.Join(tmpDir, filename)
+		exists := false
+		configData, exists = tryReading(t, path)
+		if exists {
+			configPath = path
+			break
+		}
+	}
+
+	toIgnore := ""
+
+	if configPath == "" {
+		configPath = filepath.Join(tmpDir, filenames[0])
+		toIgnore = filenames[0]
+		configData = ""
+	}
+
+	newConfigData := configData
+	var applied []string
+
+	for _, configName := range utils.SortedKeys(validConfig) {
```

Comment:
The keys are the "path" in YAML to apply the override to (e.g. `config2.resources.jobs.example_job`)?

If so, please include comments or examples of the shape of the input to this function to clarify.